README.md

```diff
@@ -4,67 +4,47 @@ Lightweight Javascript utility for using Artsy's Gemini service to upload direct
 
 ## Example
 
-Add to your script tags
+Install with Yarn/NPM:
```

Comment:
Might be worth promoting these yarn/npm instructions up to an `## Installation` section above `## Example`.